packages/template-retail-react-app/app/components/passwordless-login/index.jsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React, {useState} from 'react'
+import PropTypes from 'prop-types'
+import {FormattedMessage} from 'react-intl'
+import {Button, Divider, Stack, Text} from '@salesforce/retail-react-app/app/components/shared/ui'
+import LoginFields from '@salesforce/retail-react-app/app/components/forms/login-fields'
+import {noop} from '@salesforce/retail-react-app/app/utils/utils'
+import StandardLogin from '../standard-login/index'
+import SocialLogin from '@salesforce/retail-react-app/app/components/social-login'
+
+const PasswordlessLogin = ({form, idps = [], clickForgotPassword = noop}) => {
```

Comment:
You're passing in idps here. Good for dependency injection, but isn't consistent with how you are doing it  in the above file (per my previous comment) .. we should choose to be consistent.